content/en/docs/Config/webhook.md

```diff
@@ -50,4 +51,46 @@ Then, in Argus' `config.yml`
           CUSTOM_HEADER_NAME: hello123
 ```
 
-would execute ["SOMETHING.SH" "hello123"]
\ No newline at end of file
+would execute ["SOMETHING.SH" "hello123"]
+
+## Using GitLab Webhooks
+GitLab webhooks can be used to start GitLab CI pipelines through which the applications are updated.
+For this, a `Pipeline trigger` must be created in GitLab in the project settings under `CI/CD`.
+
+[Offical Docs](https://docs.gitlab.com/ee/ci/triggers/#use-a-webhook)
+
+The `token` must be stored as secret and doesn't have to be added to the URL. 
+
+Example URL: `https://gitlab.com/api/v4/projects/<project_id>/ref/<ref_name>/trigger/pipeline`
+
+config.yaml
+```yaml
+webhook:
+  ...
+  # As many of these (below) as you like, just ensure they have unique ID's
+  EXAMPLE_WEBHOOK_ID:
+    type: gitlab                # Type of WebHook to send
+    url: https://WEBHOOK_URL    # WebHook URL to send to
+    allow_invalid_certs: false  # Accept invalid HTTPS certs/not
+    secret: SECRET              # WebHook Token
+    desired_status_code: 201    # Status code to use indicating a success. Using 0 will accept any 2XX status code
+    delay: 0s                   # Delay before sending the webhooks when a new release is found
+                                # (only used when `auto_approve` is true for the service)
+    max_tries: 3                # Maximum number of times to try sending this message until a send is successful
+    silent_fails: false         # Whether to notify the service's notifiers if max_tries fails occur
+```
+
+## Custom variables
```

Comment:
Could we make this and custom_headers above ### so that they're under the GitLab and GitHub sections please?

Reply:
Done